src/socket.coffee

```diff
@@ -55,7 +55,7 @@ createSocket = (id, frame) ->
   checks = 0
   check = =>
     # send test message NO ENCODING
-    frame.postMessage [id, XD_CHECK, {}], "*"
+    try frame.postMessage [id, XD_CHECK, {}], "*"
```

Comment:
The fix is simple, I just added a try at here, to avoid the crashing problem caused by invalid iframe's window.